client/my-sites/checkout/composite-checkout/components/item-variation-picker/item-variation-dropdown.tsx

```diff
@@ -1,91 +1,10 @@
-import { Gridicon } from '@automattic/components';
-import { css } from '@emotion/react';
-import styled from '@emotion/styled';
+import { CustomSelectControl } from '@wordpress/components';
 import { useTranslate } from 'i18n-calypso';
-import { FunctionComponent, useCallback, useEffect, useState } from 'react';
+import { FunctionComponent } from 'react';
 import { ItemVariantPrice } from './variant-price';
-import type { ItemVariationPickerProps, WPCOMProductVariant } from './types';
-import type { ResponseCartProduct } from '@automattic/shopping-cart';
+import type { ItemVariationPickerProps } from './types';
 
-interface CurrentOptionProps {
```

Comment:
Current implementation has been replaced by an instance of `CustomSelectControl`.